ci-operator/step-registry/dpf/hypervisor/deploy-cluster/dpf-hypervisor-deploy-cluster-commands.sh

```
@@ -136,30 +136,32 @@ else
-  echo "ERROR: File env.user_${CLUSTER_NAME} file does not exist: ${REMOTE_MAIN_WORK_DIR}/env/env.user_${CLUSTER_NAME}"
+# Generate the .env file using user-env from Vault cluster profile
+USER_ENV_FILE="${CLUSTER_PROFILE_DIR}/user-env"
+echo "Verifying user-env file from Vault cluster profile exists..."
+if [[ ! -f "${USER_ENV_FILE}" ]]; then
+  echo "ERROR: user-env file not found at ${USER_ENV_FILE}"
   exit 1
 fi
 
-echo "File ${REMOTE_MAIN_WORK_DIR}/env/env.user_${CLUSTER_NAME} was found on hypervisor"
+echo "Copying user-env from Vault cluster profile to hypervisor..."
+if ! scp ${SSH_OPTS} "${USER_ENV_FILE}" root@${REMOTE_HOST}:${REMOTE_WORK_DIR}/openshift-dpf/user.env; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)